src/components/AssignedUsers/utils.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-prototype-builtins */
+import { differenceBy } from 'lodash';
+
+// eslint-disable-next-line import/prefer-default-export
+export function findObjectDifferences(prevUsers = [], newUsers = []) {
```

Comment:
To be honest, it seems to me that this is not quite the right name for this particular function, since in reality it does not return an object difference (unlike the [objectDifference.js](https://github.com/folio-org/stripes-acq-components/blob/master/lib/utils/objectDifference/objectDifference.js#L39)).

Reply:
You are right. I have improved the function name